spec/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/refresher_spec.rb

```diff
@@ -127,6 +127,18 @@ def assert_specific_vm
         :version      => "AIX 7.1, 7100-05-05-1939"
       )
 
+      expect(vm.advanced_settings.find {|setting| setting['name'] == 'entitled_processors'}).to have_attributes(
```

Comment:
Rubocop is right here, should be `{ |setting| setting['name'] == 'entitled_processors' }` and same with the next two